service/src/main/kotlin/io/provenance/explorer/config/interceptor/JwtInterceptor.kt

```diff
@@ -0,0 +1,46 @@
+package io.provenance.explorer.config.interceptor
+
+import io.provenance.explorer.OBJECT_MAPPER
+import io.provenance.explorer.domain.extensions.fromBase64
+import org.springframework.http.HttpHeaders
+import org.springframework.web.servlet.HandlerInterceptor
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+class JwtInterceptor : HandlerInterceptor {
+
+    companion object {
+        const val X_ADDRESS = "x-address"
+        const val X_PUBLIC_KEY = "x-public-key"
```

Comment:
These headers are too generic and need to include the entity they represent.

`x-FOO-address`
`x-FOO-public-key`
`x-FOO-address-public-key` (if it belongs to the address)

Example:
`x-pio-address`
`x-pio-address-public-key`